### PR TITLE
[67]  Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by gracioust**
## Task: `Style the Product Details Page using CSS`
- `Styled the Product Details Page using CSS`
product1
product2
product3
product4
product5
product6

### CODE REVIEW

**Comment by adaugochi** [comment]
I expect the related product item to appear one on a mobile screen, not two

**Comment by gracioust** [comment]
> I expected the related product item to appear one on a mobile screen, not two

Okay, done that

**Comment by gracioust** [comment]
**THE RELATED IMAGES HAVE BEEN ADJUSTED TO DISPLAY ONE BY ONE IN MOBILE VIEW**
product4
product5
product6
product7

File: submissions/tosinIshola/HTML/Home.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<html lang="en">
```
Comment (adaugochi) [inline submissions/tosinIshola/HTML/Home.html:1]: Which page is this? Where are ask to create a home page? If no, kindly remove it

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by gracioust** [comment]
Done please kindly help me review this PR and merge, i have been delayed enough on this task.

File: submissions/tosinIshola/html-forms/contact_form.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <img src="../HTML/Images/fashion.jpg" alt="logo" id="logo" >
     <div>
             <ul>
-               <li><a href="">HOME</a></li>  <br>
+               <li><a href="../HTML/Home.html">HOME</a></li>  <br>
```
Comment (adaugochi) [inline submissions/tosinIshola/html-forms/contact_form.html:18]: You forgot to remove this

File: submissions/tosinIshola/html-forms/signup.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
     <img src="../HTML/Images/fashion.jpg" alt="logo" id="logo" >
     <div>
             <ul>
-               <li><a href="">HOME</a></li>  <br>
+               <li><a href="../HTML/Home.html">HOME</a></li>  <br>
```
Comment (adaugochi) [inline submissions/tosinIshola/html-forms/signup.html:18]: You forgot to remove this

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/tosinIshola/html-forms/signup.html (hunk 3 above)
Comment (gracioust) [reply submissions/tosinIshola/html-forms/signup.html:18]: Done

File: submissions/tosinIshola/html-forms/contact_form.html (hunk 2 above)
Comment (gracioust) [reply submissions/tosinIshola/html-forms/contact_form.html]: Done

**Review by adaugochi — APPROVED** [review]

**Pull request merged**